ivy/functional/backends/torch/experimental/linear_algebra.py

```diff
@@ -91,15 +91,16 @@ def diagflat(
     return ret
 
 
+diagflat.unsupported_dtypes = ("float16",)
```

Comment:
This should be handled by the `with_unsupported_dtypes` decorator so You can revert these changes in this file